SharkyCTF2020/Web/Containment.md

```diff
@@ -0,0 +1,34 @@
+# Containment Forever
+
+> Hello, welcome on "Containment Forever"! There are 2 categories of posts, only the first is available, get access to the posts on the flag category to retrieve the flag.

```

Comment:
There are 2 categories of posts. Only the posts in the first category are available. If we manage to get access to the posts on the flag category, it is highly probable we will retrieve the flag.

Reply:
This line is the description of the challenge they gave on the CTFd, it's just copy paste :)